src/pages/index.astro

```diff
@@ -4,15 +4,38 @@ import "../index.scss";
 ---
 
 <Layout title="Welcome to Astro.">
-  <main>
-<section class="hero container">
-  <div class="hero__content">
-    <h1 class="hero--title">Travel without limits!</h1>
-      <p class="hero--description">
-        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Soluta ut facere aliquam impedit ullam minusreprehenderit quia in laudantium sint.</p>
-       <button class="hero--cta">Check offer</button>
-  </div>
-    </section>
-  </main>
+  <body>
+    <main>
+      <section class="hero container">
+        <div class="hero__content">
+          <h1 class="hero--title">Travel without limits!</h1>
+          <p class="hero--description">
+            Lorem, ipsum dolor sit amet consecte tur adipisicing elit. Soluta ut
+            facere aliquam impedit ullam minusreprehenderit quia in laudantium
+            sint.
+          </p>
+          <button class="hero--cta">Check offer</button>
+        </div>
+        <div id="modal" class="modal">
+          <button class="modal-icon--close">
+            <img src="../cross.svg" alt="cross icon" />
+          </button>
+          <h2 class="modal--title">Modal title</h2>
+          <p class="modal--description">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Tempore ea
+            consequuntur cupiditate sequi. Ipsa iste dignissimos, odit natus
+            nisi quod ipsam dolorum quae temporibus similique expedita error ad
+            quo corporis ipsum ea cumque provident, accusantium quia
+            perspiciatis? Id pariatur eligendi ex nisi quos magnam non ipsum
+            odit beatae! Sint, voluptatum?
+          </p>
+          <div class="modal__btns">
+            <button class="modal-btn modal-btn--close">close</button>
+            <button class="modal-btn modal-btn--confirm">confirm</button>
+          </div>
```

Comment:
Please add divs to wrap:
- modal itself (call it `modal-wrapper`
- modal header
- modal content
- modal footer

Reply:
Added divs, but I'm not sure where I should put button with cross.svg icon. I placed it in 'modal-header'.

Reply:
It can be in the header or above. We can leave it's placement as is.